src/AcceptanceTests/AcceptanceTests/ConfigureEndpointLearningPersistence.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.AcceptanceTesting.Support;
+using NUnit.Framework;
+
+public class ConfigureEndpointLearningPersistence
+{
+    public Task Configure(string endpointName, EndpointConfiguration configuration, RunSettings settings, PublisherMetadata publisherMetadata)
+    {
+        var testRunId = TestContext.CurrentContext.Test.ID;
+
+        string tempDir;
+
+        if (Environment.OSVersion.Platform == PlatformID.Win32NT)
+        {
+            //can't use bin dir since that will be too long on the build agents
+            tempDir = @"c:\temp";
+        }
+        else
+        {
+            tempDir = Path.GetTempPath();
+        }
+
+        storageDir = Path.Combine(tempDir, testRunId);
+
+        configuration.UsePersistence<InMemoryPersistence, StorageType.Subscriptions>();
```

Comment:
The learning transports doesn't need any storage for timeouts and subscriptions?

Reply:
@andreasohlund That's just a copy & past from core: https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.AcceptanceTests/ConfigureEndpointLearningPersistence.cs

Reply:
I've removed the persistence config entirely.

Reply:
@bording turns out that the core needs those lines for the tests that use the FakeTransport eg. https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.AcceptanceTests/Core/FakeTransport/CriticalError/When_registering_custom_critical_error_handler.cs#L39

So core needs them  but downstream should be ok to not use